website/i18n/ru/docusaurus-plugin-content-docs/current/usage/methods.md

```diff
@@ -112,7 +112,78 @@ class SomeApiService::Posts::Create < SomeApiService::Base
 end
 ```
 
-## Сокращения для методов
+## Группа из нескольких методов
+
+Собрать в одну группу выполнение несколько методов можно при помощи метода `stage`.
```

Comment:
You can use the `stage` method to group multiple methods into one execution group.